index.js

```diff
@@ -12,25 +12,15 @@ var net = require("net"),
     parsers = [], commands,
     connection_id = 0,
     default_port = 6379,
-    default_host = "127.0.0.1";
-
-// can set this to true to enable for all connections
-exports.debug_mode = false;
-
-var arraySlice = Array.prototype.slice;
-function trace() {
-    if (!exports.debug_mode) return;
-    console.log.apply(null, arraySlice.call(arguments));
-}
+    default_host = "127.0.0.1",
+    debug = util.debuglog('redis');
```

Comment:
This unfortunately won't work on node 0.10 or lower, util.debuglog was added in 0.11

See https://github.com/brycebaril/client-request/blob/master/request.js#L15-L19 for what I did in another library to work around that